en/docs/administer/monitoring-logs.md

```diff
@@ -29,11 +29,7 @@ the `          <IS_HOME>/repository/logs         ` directory.
         mentioned scenario to plug the handlers and appenders. The following
         default log4j2 appenders in the
         `            log4j2.properties           ` file are used for this
```

Comment:
If we are removing the content mentioning the appenders, we should remove this sentence from the content as well right? 

"The following default log4j2 appenders in the  `            log4j2.properties           ` file are used for this"

Reply:
done